frontend/src/services/mock.ts

```diff
@@ -1,13 +1,28 @@
-import {User} from "../types/user";
+import {User, UserType} from "../types/user";
+import {IWallet} from "../types/wallet";
 
+// test configs
 export const mockEnabled = true;
+export const mockUserType = UserType.Merchant;
+//
 
-export const mockToken = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyVHlwZSI6IkNVU1RPTUVSIiwidXNlcklkIjo3OSwiZW1haWwiOiJ1c2VydGVzdEBtYWlsLnJ1Iiwic3ViIjoidXNlcnRlc3RAbWFpbC5ydSIsImlhdCI6MTcxNDI5MTkxM30.AyL1LnHCeIr78ch5owlWw57aMFbG_4Z0c69H_-3n-jw";
+const mockTokenMerchant = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyVHlwZSI6Ik1FUkNIQU5UIiwidXNlcklkIjoxNCwiZW1haWwiOiJ1c2VydGVzdDEwQG1haWwucnUiLCJzdWIiOiJ1c2VydGVzdDEwQG1haWwucnUiLCJpYXQiOjE3MTQ3OTUzNzV9.QtmHScrPj-G2XTJvZRcft6vjR2NJ8A360pmculfTc_0";
+const mockTokenClient = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyVHlwZSI6IkNVU1RPTUVSIiwidXNlcklkIjo5MSwiZW1haWwiOiJ1c2VydGVzdDVAbWFpbC5ydSIsInN1YiI6InVzZXJ0ZXN0NUBtYWlsLnJ1IiwiaWF0IjoxNzE0Nzk1MzI3fQ.INChxr6QbqP_xmjZf77O9vTLv3UsmjF6PcmLonL05gM";
+
+export const mockToken = mockUserType === UserType.Merchant ? mockTokenMerchant : mockTokenClient;
```

Comment:
Now, if you set `export const mockUserType = UserType.Customer;`, you will get here `TS2367: This comparison appears to be unintentional because the types UserType.Customer and UserType.Merchant have no overlap.`
because ts knows that mockUserType can only be Customer https://stackoverflow-com.translate.goog/questions/76291949/how-to-turn-off-error-ts2367-this-comparison-appears-to-be-unintentional-becaus?_x_tr_sl=en&_x_tr_tl=ru&_x_tr_hl=ru&_x_tr_pto=sc

to fix it either:
1) export const mockUserType = UserType.Customer as UserType;
2) supress error